zulipterminal/ui_tools/boxes.py

```diff
@@ -403,6 +403,23 @@ def reactions_view(self, reactions: List[Dict[str, Any]]) -> Any:
         except Exception:
             return ''
 
+    def footlinks_view(self, message_links: List[Tuple[str, str]]) -> Any:
+        if not message_links:
+            return ''
+
+        indexed_links = [
+            '{}: {}\n'.format(index, link)
+            for index, (text, link) in enumerate(message_links, start=1)
+        ]
+        indexed_links[-1] = indexed_links[-1][:-1]  # Remove the last newline.
+
+        footlinks = [
+            ('msg_footlink', indexed_link)
+            for indexed_link in indexed_links
+        ]
+        return urwid.Padding(urwid.Text(footlinks), align='left', left=10,
```

Comment:
**Before**:

![Screenshot from 2020-07-06 23-11-05](https://user-images.githubusercontent.com/43616959/86623370-16d6a780-bfdf-11ea-881a-4048acef308d.png)

**After**:

![Screenshot from 2020-07-06 23-17-20](https://user-images.githubusercontent.com/43616959/86623386-1dfdb580-bfdf-11ea-878c-fd0854b9307f.png)

I prefer the _after_ as it adds a bit of distinction, between the content and the footlinks, and gives it more width. Additionally, I don't see much interference. Thoughts?

Reply:
I'm not sure I have a strong preference at this stage, but I think this change will be easier to adjust later, and may change if we have either of
* link text & names (eg. link & google.com) styled as currently, but the numbers styled the same with the new style
* the ellipsis to avoid long links wrapping

Reply:
I agree. I have pushed the new update with all these three changes - namely _after_ alignment, new styling only for link indexes and ellipsis for now.